runtime.md

```diff
@@ -9,8 +9,11 @@ $ funC [global-options] <COMMAND> [command-specific-options] <command-specific-a
 
 ## Global options
 
-None are required, but the runtime may support options that start with at least one hyphen.
-Global options may take positional arguments (e.g. `--log-level debug`), but the option parsing must be such that `funC <COMMAND>` is unambiguously an invocation of `<COMMAND>` for any `<COMMAND>` that does not start with a hyphen (including commands not specified in this document).
+None are required, but the runtime MAY support options that start with at least one hyphen.
+Global options MAY take positional arguments (e.g. `--log-level debug`).
+Command names MUST not start with hyphens.
```

Comment:
I think the text in the PR is good and clear. 

Your statements clarified for me how you read them, specifically with the future guarantees and the purpose of making easier to parse. I don't think we need to add any statements about future revisions or the rationale for the requirement. It's the pattern selected .. so it is what it is.